astroquery/nasa_exoplanet_archive/nasa_exoplanet_archive.py

```diff
@@ -58,14 +58,21 @@ def get_confirmed_planets_table(self, cache=True, show_progress=True,
         table_path : str (optional)
             Path to a local table file. Default `None` will trigger a
             download of the table from the internet.
+        all_columns : bool (optional)
+            Return all available columns. The default returns only the
+            columns in the default category at the link above.
         Returns
```

Comment:
could you add an extra empty line here, the docstring doesn't render correctly atm.